support/cas-server-support-saml-idp-discovery/src/main/java/org/apereo/cas/services/SamlIdentityProviderDiscoveryFeedService.java

```diff
@@ -0,0 +1,42 @@
+package org.apereo.cas.services;
+
+import org.apereo.cas.entity.SamlIdentityProviderEntity;
+import org.apereo.cas.web.DelegatedClientIdentityProviderConfiguration;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.Collection;
+
+
+
+/**
+ * This is {@link SamlIdentityProviderDiscoveryFeedService}.
+ *
+ * @author Misagh Moayyed
+ * @author Sam Hough
+ * @since 6.6.0
+ */
+public interface SamlIdentityProviderDiscoveryFeedService {
+
+    /**
```

Comment:
This is not valid javadoc. You should explain the intention of the method, albeit briefly, same with the other method here.

Reply:
OK, I tried looking for a good example in the code base. If you know of a canonical example of hand that would be very helpful. Part of the problem is I'm just re-arranging somebody else's code so I'm guessing intent. I'll try again.

Reply:
This is proper javadoc:

<img width="1056" alt="image" src="https://user-images.githubusercontent.com/1205228/162382010-4a654893-abba-4e10-8d24-1a4ae0c8d930.png">

You will find this sort of structure everywhere. It not that your notes here are inadequate. It's that the javadoc tool will fail to accept this because it's invalid. Whether you want to write 3 paragraphs or 3 words is up to you.

Reply:
OK. Thanks. I think I get you. I struggle with the /** gets the foo @return the foo */ Foo getFoo() ... code ;) I'll try again and run javadoc task this time.

Reply:
OK. I think I've pushed a fix for those. I ran checkstyle and (I believe) javadoc without error.